web/client/plugins/ResourcesCatalog/components/ResourceCard.jsx

```diff
@@ -402,7 +412,8 @@ const ResourceCard = forwardRef(({
     const {
         icon,
         viewerUrl,
-        thumbnailUrl
+        thumbnailUrl,
+        contextMapWithoutContextPermission
     } = getResourceInfo(resource);
```

Comment:
In addition to the previous comment we should use status object instead of info
https://github.com/geosolutions-it/MapStore2/blob/e6f0797cf03a6268dd6ae4296ce34e53b6ce63ce/web/client/utils/GeostoreUtils.js#L102-L120

If we want we could use the already available items that are icon visible on the grid card but we could also expose new property e.g.:

```js
{
  cardClassNames: ['ms-broken-resource-card'], // based on condition
  cardTooltipId: 'resourcesCatalog.warningForContextMapWithoutContextPermssion' // based on condition
  items: [...]
}
```

if we include something `cardClassNames` and `cardTooltipId` we can avoid to have hardcoded inline style and too specific name for props inside the generic component.